spidy/py_messages/cap.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import roslib
+#roslib.load_manifest('my_package')
+import sys
+import rospy
+import cv2
+from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+
+class image_converter:
+
+  def __init__(self):
+
+    self.bridge = CvBridge()
+    self.image_sub = rospy.Subscriber("mybot/camera1/image_raw",Image,self.callback)
+    self.counter=0
+  def callback(self,data):
+    try:
+      cv_image = self.bridge.imgmsg_to_cv2(data, "bgr8")
+      cv2.imwrite("/home/mostafa/Documents/my_ws/src/spidy/saved_images/image"+str(self.counter)+'.png',cv_image)
```

Comment:
hardcode that will not work on my laptop

Reply:
now it is solved